ios/Podfile

```diff
@@ -1,4 +1,4 @@
-platform :ios, '12.0'
```

Comment:
Add this on the first line ([docs](https://capacitorjs.com/docs/updating/4-0#raise-ios-deployment-target))

```
require_relative '../../node_modules/@capacitor/ios/scripts/pods_helpers'
```

also add this block on the last line
```
post_install do |installer|
  assertDeploymentTarget(installer)
end
```